### ISSUE TITLE
Is it possible to get iPTM for pairs within a multimer?

### ISSUE THREAD

**Issue by Abhishaike** [sokrypton/ColabFold#353]
I'm looking to calculate iPTM for each pair of proteins in a given complex (using the below formula from the Alphafold-Multimer paper). To do this, I believe I would need the logits, breaks, and bin representations from Alphafold...which it doesn't seem like Colabfold currently gives? Is it possible to access this information?

<img width="722" alt="Screen Shot 2023-01-12 at 1 47 57 PM" src="https://user-images.githubusercontent.com/19715304/212153063-e62297f1-9876-484c-9889-8b1d5656184e.png">

https://github.com/deepmind/alphafold/blob/0be2b30b98f0da7aecb973bde04758fae67eb913/alphafold/model/model.py#L48
```
confidence_metrics['iptm'] = confidence.predicted_tm_score(
          logits=prediction_result['predicted_aligned_error']['logits'],
          breaks=prediction_result['predicted_aligned_error']['breaks'],
          asym_id=prediction_result['predicted_aligned_error']['asym_id'],
          interface=True)
```

This is the function it's calling:
https://github.com/deepmind/alphafold/blob/0be2b30b98f0da7aecb973bde04758fae67eb913/alphafold/common/confidence.py#L111

**Comment by Abhishaike** [comment]
Would be more than happy to contribute this to colabfold batch, please let me know if that'd be desirable!

**Comment by sokrypton** [comment]
I thought we already returned this as one of the outputs...

**Comment by Abhishaike** [comment]
iPTM is returned, but in the case of multimeric proteins with **more** than 2 chains, there isnt an iptm for each of the pairs of proteins
